Add FAQ and Troubleshooting Section to FB CAPI (Actions) doc (#2461)
segmentio/segment-docs: src/connections/destinations/catalog/actions-facebook-conversions-api/index.md

If you choose this option, each source sends different events, and deduplication is not necessary.

## FAQ & Troubleshooting

### Other Standard Events

> If you want to send a [Facebook standard event](https://developers.facebook.com/docs/meta-pixel/reference#standard-events){:target="_blank"} that Segment does not have a prebuilt mapping for you can use the [Custom Event action](/docs/connections/destinations/catalog/actions-facebook-conversions-api/#custom-event) to send the standard event. For example, if you want to send a `CompleteRegistration` event, create a mapping for Custom Event, set up your Event Trigger criteria for completed registrations, and input a literal string of "CompleteRegistration" as the Event Name. You can use the Custom Data key/value editor to add fields that are in the `CompleteRegistration` event such as `content_name` and `currency`.

Reviewer: ```suggestion
If you want to send a [Facebook standard event](https://developers.facebook.com/docs/meta-pixel/reference#standard-events){:target="_blank"} that Segment does not have a prebuilt mapping for, you can use the [Custom Event action](/docs/connections/destinations/catalog/actions-facebook-conversions-api/#custom-event) to send the standard event. For example, if you want to send a `CompleteRegistration` event, create a mapping for Custom Event, set up your Event Trigger criteria for completed registrations, and input a literal string of "CompleteRegistration" as the Event Name. You can use the Custom Data key/value editor to add fields that are in the `CompleteRegistration` event such as `content_name` and `currency`.
```

### PII Hashing

> Segment will hash the following fields using a SHA-256 hash:

Reviewer 2: ```suggestion
Segment creates a SHA-256 hash of the following fields:
```

- External ID
- Email
- Phone
- Gender
- Data of Birth
- Last Name
- First Name
- City
- State
- Zip Code
- Country

> If you use the Facebook Pixel, the Pixel library also hashes the External ID. This means External IDs will match across Facebook Pixel and Facebook Conversions API if they use External ID for [deduplication](https://developers.facebook.com/docs/marketing-api/conversions-api/deduplicate-pixel-and-server-events/#fbp-or-external-id){:target="_blank"}.

Reviewer: ```suggestion
If you use the Facebook Pixel, the Pixel library also hashes the External ID. This means External IDs will match across Facebook Pixel and the Facebook Conversions API if they use the External ID for [deduplication](https://developers.facebook.com/docs/marketing-api/conversions-api/deduplicate-pixel-and-server-events/#fbp-or-external-id){:target="_blank"}.
```

Author: I don't think we need a "the" before Facebook Conversions API since we don't do that elsewhere in the doc unless we are talking about the destination (vs the actual API).

> If you are using the Facebook Pixel, the Pixel library will also hash the External ID. This means External IDs will match across Facebook Pixel and Facebook Conversions API if are relying External ID for [deduplication](https://developers.facebook.com/docs/marketing-api/conversions-api/deduplicate-pixel-and-server-events/#fbp-or-external-id){:target="_blank"}.

Reviewer 2: ```suggestion
If you use the Facebook Pixel, the Pixel library also hashes the External ID. This means External IDs will match across Facebook Pixel and Facebook Conversions API if they yse External ID for [deduplication](https://developers.facebook.com/docs/marketing-api/conversions-api/deduplicate-pixel-and-server-events/#fbp-or-external-id){:target="_blank"}.
```
